pkg/env/env.go

```diff
@@ -62,6 +68,8 @@ func InitConfig() error {
 		logrus.Debug("Can't read config file, all values will be prompted, err: ", err)
 	}
 
+	getNestedArgValues()
```

Comment:
IIRC, this function is Ok to implement as long as you can't do "unmarshal" from config here. We did this in past, and I don't have a time to look into. I hope you evaluated this option.

Otherwise, entire PR looks good to me.

Reply:
We can't get nested values in the same way as `output-dir` or `source`.